libnavigation-core/src/test/java/com/mapbox/navigation/core/MapboxNavigationTest.kt

```diff
@@ -127,4 +172,97 @@ class MapboxNavigationTest {
 
         verify(exactly = 1) { tripSession.unregisterAllVoiceInstructionsObservers() }
     }
+
+    @Test
+    fun fasterRoute_timerStarted() {
```

Comment:
Minor thing

Normally is good to structure the tests following the triple A 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40 http://wiki.c2.com/?ArrangeActAssert
If the `A`rrange part gets too long we should create factory or builder methods.

This applies to the rest of the tests 👇

Reply:
NIT This is still valid